nixos/modules/services/misc/home-assistant.nix

```diff
@@ -251,16 +251,54 @@ in {
         User = "hass";
         Group = "hass";
         Restart = "on-failure";
+        KillSignal = "SIGINT";
+
+        # Hardening
+        AmbientCapabilities = lib.mkForce [
```

Comment:
Is it feasible to only add these if said module is actually enabled?

Reply:
Yeah, could be. The `lib.mkForce` is a Copy/paste remainder of my local test setup.

Reply:
Done in [`4a21fd3` (#120526)](https://github.com/NixOS/nixpkgs/pull/120526/commits/4a21fd352959cc5aca413869f671b46366e22d13).